apps/mull-api/src/app/event/location-autocomplete/location-autocomplete.resolver.ts

```
@@ -0,0 +1,14 @@
+import { Query, Args } from '@nestjs/graphql';
+import axiosInstance from '../../../../axiosConfig';
+
+export class LocationAutocompleteResolver {
+  @Query(/* istanbul ignore next */ () => [String])
+  async getAutocompletedLocations(@Args('userInput') userInput: String) {
```

This should throw an error when something goes wrong. For example, I forgot to add in the key in my `.env` file, and it took me a while to understand why

---

Thank you for implementing my suggestion @JimLau49